tox.ini

```
@@ -100,4 +100,4 @@ skip_install = true
 changedir = .
 description = check code style, e.g. with flake8
 deps = flake8
-commands = flake8 specutils --count --max-line-length=100 --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
+commands = flake8 specutils --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
```

```suggestion
commands =
    flake8 specutils -qq --statistics --select=E501,W505
    flake8 specutils --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
```
Assuming we don't want to _enforce_ line lengths anytime soon, this might be helpful to keep track of how much of the code conforms (or not).

---

Sure, but changing this won't help your editor; this is just for the CI. Your editor only picks up `setup.cfg`.